website/README.md

```diff
@@ -24,6 +24,26 @@ If you are not familiar with Jekyll, be sure to check out its
 [documentation][jekyll-docs]. It is recommended to follow the provided
 step-by-step tutorial.
 
+###### Switching Ruby versions
+
+The required Ruby version is set in `.ruby-version`. To switch, you can use
+[rvm][rvm] to manage your Ruby version.
+
+###### Resolving Bundler issues
+
+On Macos you may get an error such as the following when `bundle install`:
+
+```sh
+fatal error: 'openssl/ssl.h' file not found
+```
+
+In that case, run
+
+```sh
+bundle config build.eventmachine --with-cppflags="-I$(brew --prefix openssl)/include"
+bundle install
+```
+
 # Deployment
```

Comment:
Based on the other headers, shouldn't this be `deploying`?

Reply:
Good one; I think so.